modules/ipi-install-network-requirements.adoc

```diff
@@ -147,16 +147,16 @@ Define a consistent clock date and time format in each cluster node's BIOS setti
 You can reconfigure the control plane nodes to act as NTP servers on disconnected clusters, and reconfigure worker nodes to retrieve time from the control plane nodes.
 
 [discrete]
-== State-driven network configuration requirements (Technology Preview)
+== State-driven network configuration requirements
 
-{product-title} supports additional post-installation state-driven network configuration on the secondary network interfaces of cluster nodes using `kubernetes-nmstate`. For example, system administrators might configure a secondary network interface on cluster nodes after installation for a storage network.
+Installer-provisioned installation supports additional post-installation state-driven network configuration on the secondary network interfaces of cluster nodes using `kubernetes-nmstate`. For example, system administrators might configure a secondary network interface on cluster nodes after installation for a storage network.
```

Comment:
Why are we mixing IPI into this topic when all of this is happening post-installation.
As this is about post-installtion, I don't think we need to mention anythin in relation to the installers, do we?

Reply:
Agreed, the fact that it's IPI or UPI shouldn't matter if this is all taken care of post-deployment. The same mechanisms for applying such configuration (either via kubernetes-nmstate or MachineConfig) should be available, regardless of the deployment mechanism.